src/main/webapp/dashboard.jsp

```diff
@@ -11,28 +11,47 @@
 %>
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>TravelPal Dashboard</title>
-    <link rel="stylesheet" href="CSS/dashboard.css">
-</head>
-<body>
-    <header>
-        <h1>TravelPal Dashboard</h1>
-        <h3>Welcome <%= userEmail %>!</h3>
-    </header>
-
-    <nav>
-        <ul>
-            <li><a href="createTrip.jsp">Create Trip</a></li>
-            <li><a href="currencyConv.jsp">Currency Converter</a></li>
-        </ul>
-    </nav>
-
-    <footer>
-        <p>&copy; Fall 2024 Team 6 TravelPal. All rights reserved.</p>
-    </footer>
-</body>
+	<head>
+	    <meta charset="UTF-8">
+	    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	    <title>TravelPal Dashboard</title>
+	    <link rel="stylesheet" href="CSS/dashboard.css">
+	    
+	    <script>
```

Comment:
In-line JavaScript is fine here, but might be better if it goes into a separate file when there are more JavaScript code.